godot/src/utils/Promise.gd

```diff
@@ -156,6 +156,8 @@ class RaceAwaiter:
 # Each function in the array is expected to be a coroutine or a function
 # that returns a promise.
 static func co_all(funcs: Array) -> Array:
+	if funcs.is_empty():
+		return []
```

Comment:
Thanks. I'm going to change this at https://github.com/kuruk-mm/gdscript-async-utils